src/ensembl/src/content/app/species/state/general/speciesGeneralHelper.ts

```diff
@@ -660,38 +661,53 @@ export const getStatsForSection = (props: {
       })
     : undefined;
 
+  const isExpandedContentSame = isEqual(
+    Object.keys(filteredData).sort(),
+    summaryStatsKeys?.sort()
+  );
```

Comment:
Just to confirm, why do we need to sort and check for equality?

Reply:
that should also work. This will do a strict check (just in case if the content are different)

Reply:
decided to go with .length() check